src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -225,7 +225,8 @@ public void setCredentialsId(@CheckForNull String credentialsId) {
     @DataBoundSetter
     public void setTraits(@CheckForNull List<SCMTrait<? extends SCMTrait<?>>> traits) {
```

Comment:
the new SCM api prevents using reduced generics on the setter, however that should be fine.